kubevirt_scripts/bootfromcdonce.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+#### IMPORTANT: This script is only meant to show how to implement required scripts to make custom hardware compatible with FakeFish. Dell hardware is supported by the `idrac-virtualmedia` provider in Metal3.
+#### This script has to set the server's boot to once from cd and return 0 if operation succeeded, 1 otherwise
+#### You will get the following vars as environment vars
+#### BMC_ENDPOINT - Has the BMC IP
+#### BMC_USERNAME - Has the username configured in the BMH/InstallConfig and that is used to access BMC_ENDPOINT
+#### BMC_PASSWORD - Has the password configured in the BMH/InstallConfig and that is used to access BMC_ENDPOINT
+
+echo "Not supported on KubeVirt. Node will boot from ISO if primary disk is empty."
+
+if [ $? -eq 0 ]; then
```

Comment:
I'm not sure what this code does, `echo` will return 0

Reply:
Right.